test/module.js

```diff
@@ -398,10 +401,9 @@ const cases = {
   'bg-left-bottom': { 'background-position': 'left bottom' },
   'bg-right-bottom': { 'background-position': 'right bottom' },
 
-  'bg-primary': { 'background-color': theme.colors['primary'] },
-  'bg-red': { 'background-color': theme.colors['red']['default'] },
-  'bg-red-100': { 'background-color': theme.colors['red']['100'] },
-  'bg-gray-500': { 'background-color': theme.colors['gray']['500'] },
+  'bg-red': { 'background-color': 'rgba(255,0,0,var(--ow-bg-opacity,1))' },
+  'bg-red-100': { 'background-color': 'rgba(255,245,245,var(--ow-bg-opacity,1))' },
+  'bg-gray-500': { 'background-color': 'rgba(160,174,192,var(--ow-bg-opacity,1))' },
```

Comment:
I would like to ask, what's the point in testing `bg-red-100` & `bg-gray-500` when further up in the file `bg-red-999` & `bg-red-500` are already tested? Seems like testing the same thing. Didn't want to straight up remove in case there was something I am missing.

Reply:
Good question! So `bg-red-999` is testing a custom theme defined color, the `red-100` and `gray-500` are probably not required 😅 we probably should co-locate all the bg tests!